Formula/xinit.rb

```diff
@@ -0,0 +1,152 @@
+class Xinit < Formula
+  desc "Start the X Window System server"
+  homepage "https://gitlab.freedesktop.org/xorg/app/xinit"
+  url "https://www.x.org/releases/individual/app/xinit-1.4.1.tar.bz2"
+  sha256 "de9b8f617b68a70f6caf87da01fcf0ebd2b75690cdcba9c921d0ef54fa54abb9"
+  license all_of: ["MIT", "APSL-2.0"]
+
+  depends_on "pkg-config" => :build
+  depends_on "tradcpp"    => :build
+
+  depends_on "libx11"
+  depends_on "lndir"
+  depends_on "mkfontscale"
+  depends_on "xauth"
+  depends_on "xmodmap"
+  depends_on "xrdb"
+  depends_on "xserver"
+  depends_on "xterm"
+
+  on_macos do
+    depends_on "quartz-wm"
+
+    def caveats
+      privileged_plist = "#{plist_name.chomp ".startx"}.privileged_startx.plist"
+      <<~EOS
+        To use Homebrew's X11 as the default server, you need to run:
+          `brew services start xinit`
+        Note that doing this may cause the `Xquartz` to fail to launch.
+        For more info, see https://www.xquartz.org/FAQs.html
+        To enable the service `privileged_startx`, you need to run
+          `sudo cp #{opt_prefix}/#{privileged_plist} /Library/LaunchDaemons/`
+        and take root:admin ownership of some xinit paths:
+          #{opt_libexec}
+          #{opt_libexec}/privileged_startx
+          #{opt_prefix}
+          #{opt_bin}
+          #{HOMEBREW_PREFIX}/var/homebrew/linked/xinit
+        Then bootstrap the service:
+          `sudo launchctl bootstrap system /Library/LaunchDaemons/#{privileged_plist}`
+        This will require manual removal of these paths using `sudo rm` on
+        brew upgrade/reinstall/uninstall.
+      EOS
+    end
+  end
+
+  resource "font_cache" do
+    url "https://raw.githubusercontent.com/XQuartz/XQuartz/XQuartz-2.8.1/base/opt/X11/bin/font_cache"
+    sha256 "8f27cf55e5053686350fce6ea1078e314bd5b2d752e9da1b9051541e643e79d6"
+  end
+
+  resource "98-user.sh" do
+    url "https://raw.githubusercontent.com/XQuartz/XQuartz/XQuartz-2.8.1/base/opt/X11/etc/X11/xinit/xinitrc.d/98-user.sh"
+    sha256 "b417aea949931b7c03133535c3b5146b9403b8c3482a1c1d0a5dc01c07876c84"
+  end
+
+  def install
+    configure_args = std_configure_args + %W[
+      --with-bundle-id-prefix=homebrew.mxcl
+      --with-launchagents-dir=#{prefix}
+      --with-launchdaemons-dir=#{prefix}
+      --with-xserver=#{Formula["xserver"].opt_bin}/X
```

Comment:
Use `#{HOMEBREW_PREFIX}/bin/X` or `#{Formula["xserver"].opt_bin}/X`? The previous form is friendly to `brew unlink xserver`.

Reply:
Why not just 
```suggestion
      --with-xserver=#{Formula["xserver"].bin}/X
```

Reply:
> Why not just

Fixed.